microsoft-edge/devtools-guide-chromium/whats-new/2021/09/devtools.md

```diff
@@ -0,0 +1,148 @@
+title: What's new in DevTools (Microsoft Edge 94)
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 09/03/2021
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, f12 tools, devtools
+---
+# What's New In DevTools (Microsoft Edge 94)
+
+[!INCLUDE [note about What's New announcements from the Microsoft Edge DevTools team](../../includes/edge-whats-new-note.md)]
+
+
+<!-- 1 -->
+<!-- ====================================================================== -->
+## Search Console Error
+
+<!-- Title: Debug console errors with our new search feature -->
+<!-- Subtitle: Try search console error for a seamless path to your error's solution. -->
+
+:::image type="icon" source="../../media/2020/06/experimental-tag-14px.msft.png":::  
+
+Search the web for your **Console** error strings, from within DevTools.  When you hover over a magnifying glass icon on an error message in the **Console**, the tooltip is displayed: **Search for this message on the Web**.
+
+:::image type="complex" source="../../media/2021/09/search-console-icon.msft.png" alt-text="The Search Console Error magnifying glass icon in the Console pane" lightbox="../../media/2021/09/search-console-icon.msft.png":::
+   The **Search Console Error** magnifying glass icon in the **Console** pane
+:::image-end:::
+
+When you select the magnifying glass icon, a new tab opens in the browser and shows search results for the error string.
+
+:::image type="complex" source="../../media/2021/09/search-console-new-tab.msft.png" alt-text="New tab that opened from the Search Console Error feature" lightbox="../../media/2021/09/search-console-new-tab.msft.png":::
+   New tab that opened from the **Search Console Error** feature
+:::image-end:::
+
+For more information, navigate to [Experimental features: Search Console Error][ExpFeaturesSearchConsoleError].
+
+
+<!-- 2 -->
+<!-- ====================================================================== -->
+## DevTools extension for Visual Studio Code now with newest tools, theme support, and helpful links
+
+<!-- Title: Edge DevTools for VS Code now supports themes and uses the newest codebase -->
+<!-- Subtitle: The Edge DevTools extension for VS Code now uses the same version of the Developer Tools as your Microsoft Edge browser and we added ways to learn more and tell us what we could do better. -->
+
+The Microsoft Edge DevTools extension for Visual Studio Code now uses the newest available version of the tools and supports the themes that ship with Visual Studio Code.  The Extension sidebar now provides direct links to our documentation and to file bugs or propose enhancements.  We also added a **Close** button to DevTools, and added support for JavaScript Debugger connections on remote workspaces.
```

Comment:
That's good, but I don't like the passive voice. Instead of "the following updates or new features were released." we should use "we released the following updates or new features."

Reply:
for the record: I'm Editing the Suggestion, then clicking **Add suggestion to batch** button in PR UI.